File: Contest Solutions/Contest 4 Utilize/goodAndBad.java
import java.util.*;

public class goodAndBad {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while(t-->0){
            String s = sc.next();
            int flag = 0;
            int n = s.length() - 1;
            int count = 0;
            for(int i = 0 ; i< n ;i++){
               if(s.charAt(i) != s.charAt(i+1)){
                   count++;
                   if(count == 2){
                       flag = 1;
                       break;
                   }
                   }else{
                       count = 0;
                       continue;
                   }
               }
            if(flag == 1){
                System.out.println("Good");
            }else{
                System.out.println("Bad");
            }
            sc.close();
        }
    }
}

File: Contest Solutions/Contest 4 Utilize/goodAndBad.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int t;
    cin>>t;
    while(t--){
        string s;
        cin>>s;
        if(s.find("010")!=string::npos || s.find("101")!=string::npos)
            cout<<"Good"<<endl;
        else
            cout<<"Bad"<<endl;
    }
    return 0;
}